package.json

```diff
@@ -34,13 +34,13 @@
     "test": "npm run lint",
     "lint": "run-s lint:*",
     "lint:js": "eslint . --ext .js,.jsx,.md",
-    "lint:markdown": "markdownlint --config ./.markdownlint.json *.md ./src/content/**/*.md",
+    "lint:markdown": "markdownlint --config ./.markdownlint.json *.md ./src/content/**/*.md --ignore './src/content/**/_*.md'",
```

Comment:
Why do we need `ignore` here? Do I understand correctly that plugins and loaders docs prefixed with an underscore are the ones we're fetching?

Reply:
We need to ignore them.
Yes, with an `_` before.